package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhevmjs",
-  "version": "0.6.1",
```

Comment:
Don't bump fhevmjs, we won't publish anything. And you don't have to do bump on main.